include/deal.II/lac/scalapack.h

```diff
@@ -181,20 +179,95 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
   void invert();
 
   /**
-   * Compute all eigenvalues of a real symmetric matrix using <code>pXsyev</code>.
+   * Computing selected eigenvalues and, optionally, the eigenvectors of the real symmetric
+   * matrix \f$A \in \mathbb{R}^{M \times M}\f$.
```

Comment:
You'll have to remove the `\f` here and everywhere else. We have a script that replaces every `$` by `\f$`, so adding `\f` here and elsewhere is wrong.